src/forms/login-popup/index.jsx

```diff
@@ -160,6 +156,9 @@ const LoginForm = () => {
   };
 
   const handleSubmit = (e) => {
+    if (socialAuthnButtonRef.current) {
+      socialAuthnButtonRef.current.blur();
```

Comment:
why do we need to call blur event here?

Reply:
When the page loads, the focus is set to the social auth button. However, when we click on the submit button, the focus doesn't move to it. We made the button blur so that when the submit button is clicked, the focus shifts away from the social auth button.

Reply:
I don't understand why clicking on submit button does not shift the focus and do we need really need this line of code?

Reply:
@attiyaIshaque please remove blur event we do not need it